test/e2e-tests.sh

```diff
@@ -57,7 +57,7 @@ set -o pipefail
 all_tests=$(echo task/*/*/tests)
 
 function detect_new_tasks() {
-    git --no-pager diff --name-only "${PULL_BASE_SHA}".."${PULL_SHA}"|grep 'task/[^\/]*/[^\/]*/tests/[^/]*.yaml'|xargs dirname|sort -u
+    git --no-pager diff --name-only "${PULL_BASE_SHA}".."${PULL_PULL_SHA}"|grep 'task/[^\/]*/[^\/]*/tests/[^/]*.yaml'|xargs -I {} dirname {}|sort -u
```

Comment:
This sounds about right, but plumbing script may be buggy then: 

https://github.com/tektoncd/plumbing/blob/master/scripts/presubmit-tests.sh#L64

Reply:
I guess yes it's buggy :sweat_smile:

Reply:
feel free to send a PR to fix it in there too i guess :) 

(we don't use it because it was a pain to do this over a variable)

Reply:
Thanks will fix that in plumbing as well :sweat_smile: